op-deployer/pkg/deployer/pipeline/init.go

```diff
@@ -99,6 +91,38 @@ func Init(ctx context.Context, env *Env, _ ArtifactsBundle, intent *state2.Inten
 	return nil
 }
 
+func initCommonChecks(st *state.State) error {
+	// Ensure the state version is supported.
+	if !IsSupportedStateVersion(st.Version) {
```

Comment:
nit: I don't think `IsSupportedStateVersion` needs to be public.

Reply:
it's useful for libraries that depend on op-deployer.